app/models/principal.rb

```diff
@@ -33,7 +33,14 @@ class Principal < ActiveRecord::Base
   self.table_name = "#{table_name_prefix}users#{table_name_suffix}"
 
   has_many :members, foreign_key: 'user_id', dependent: :destroy
-  has_many :memberships, class_name: 'Member', foreign_key: 'user_id', include: [:project, :roles], conditions: "#{Project.table_name}.status=#{Project::STATUS_ACTIVE}", order: "#{Project.table_name}.name"
+  has_many :memberships, -> {
+                          includes(:project, :roles)
+                           .where(projects: { status: Project::STATUS_ACTIVE})
+                           .order('projects.name ASC')
+                           # haven't been able to produce the order using hashes
+                        },
+                        :class_name => 'Member',
```

Comment:
@myabc Would not hjave thought to see the pre 1.9 hash syntax being used by you ^^^
Every git blame in openproject leads to you, because you erased that one xD